tapfreighter/wallet.go

```diff
@@ -156,10 +156,6 @@ type AnchorVTxnsParams struct {
 	// anchored by the anchor transaction.
 	VPkts []*tappsbt.VPacket
 
-	// InputCommitments is a map from virtual package input index to its
-	// associated Taproot Assets commitment.
-	InputCommitments tappsbt.InputCommitments
```

Comment:
No longer needed as all this info is now in the `Vpkts` passed in?

Reply:
Not really. It wasn't really needed anymore once we started carrying along the passive assets explicitly. Tried to explain in the commit message:
> We always started with the input commitment, cloned it, removed the
active asset from the commitment, then added the output asset. And
finally we also committed the passive assets to that output.
But because the input commitment minus the active asset is equal to the
passive assets, we can simplify this whole procedure if we just assume
that the passive assets are in the list of the given virtual packets.